BUILD

```diff
@@ -1988,6 +1988,41 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
This seems reasonably self contained and would make a great candidate for writing some unit tests.

Reply:
This is mostly just the interface. Or, do you mean unit tests for the filter in addition to the end2end tests in https://github.com/grpc/grpc/pull/27388 ?

Reply:
:) Added